### PR TITLE
Rebases and cleans up typography

### PR DESCRIPTION
**Pull request by andresgalante**
This PR cherry picks and closes #68 to accommodate to the new global variable syntax

### CODE REVIEW

**Comment by cliffpyles** [comment]
@andresgalante Just saw a couple of minor things. Looks like the changes aren't passing the linter's tests, and there are some regressions with the button colors.

image

File: src/patternfly/components/Content/styles.scss [hunk 1]
```diff
@@ -40,143 +139,111 @@
   h5,
   h6 {
     margin: 0;
-    font-weight: $global--FontWeight--normal;
-    line-height: $global--LineHeight;
+  }
+
+  .pf-is-hero {
```
Comment (cliffpyles) [inline src/patternfly/components/Content/styles.scss]: Content is meant for raw HTML, correct?

File: src/patternfly/components/Content/styles.scss [hunk 2]
```diff
@@ -1,36 +1,135 @@
...
+      // This variable name doesn't reflect the selector, it's an excpection to the variable system.
+      margin-bottom: var(--pf-c-content--MarginBottom);
     }
   }
 
-  dl,
   ol,
   ul {
-    padding-left: $global--spacer--lg;
+    // This variable name doesn't reflect the selector, it's an excpection to the variable system.
```
Comment (cliffpyles) [inline src/patternfly/components/Content/styles.scss]: the indent on these two lines are causing the linter to fail

**Comment by andresgalante** [comment]
@cliffpyles yeah, I know, I wanted to fix those when I do css to sass variables.

File: src/patternfly/components/Content/styles.scss (hunk 2 above)
Comment (andresgalante) [reply src/patternfly/components/Content/styles.scss]: fixed it

File: src/patternfly/components/Content/styles.scss (hunk 1 above)
Comment (andresgalante) [reply src/patternfly/components/Content/styles.scss]: you are right, it isn't... it should be `pf-content__hero-title` right?

**Comment by cliffpyles** [comment]
@andresgalante Looks great!

**Pull request merged**